addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -78,3 +83,218 @@ def _get_default_payment_method_id(self):
         if self.provider != 'stripe':
             return super()._get_default_payment_method_id()
         return self.env.ref('payment_stripe.payment_method_stripe').id
+
+    # -------------------------------------------
+    # Stripe Connect
+    # -------------------------------------------
+    def _stripe_publishable_key(self):
+        """Gets Stripe Publishable Key
+
+        :returns: Stripe Publishable Key
+        :rtype: str
+        """
+        return self.stripe_publishable_key
+
+    def _stripe_secret_key(self):
+        """Gets Stripe Secret Key
+
+        :returns: Stripe Secret Key
+        :rtype: str
+        """
+        return self.stripe_secret_key
+
+    def _stripe_webhook_secret(self):
+        """Gets Stripe Webhook Secret
+
+        :returns: Stripe Webhook Secret
+        :rtype: str
+        """
+        return self.stripe_webhook_secret
+
+    def _stripe_additional_header(self, endpoint):
+        """Returns an additional header to Stripe API calls
+
+        This method adds the Stripe Account on behalf of which the Stripe call is made.
+
+        :param endpoint: the Stripe endpoint which will be called
+        :returns: An additionnal header dict
+        :rtype: dict
+        """
+        return {}
+
+    def _stripe_webhook_url(self):
+        return self.company_id.get_base_url() + '/payment/stripe/webhook'
+
+    def _stripe_create_checkout_session_values(self):
+        """Returns a dict of additional information to add in the Stripe Session
+
+        :returns: A dict of additional session information
+        :rtype: dict
+        """
+        return {}
+
+    # -------------------------------------------
+    # Stripe Create Webhook
+    # -------------------------------------------
+    def action_stripe_create_webhook(self):
+        self.ensure_one()
+        if self.stripe_secret_key and not self.stripe_webhook_secret:
+            return self._create_webhook() or True
+        return True
+
+    def _create_webhook(self):
+        """Create a webhook using the Stripe API
+        """
+        if self.stripe_webhook_secret:
+            return {
+                'type': 'ir.actions.client',
+                'tag': 'display_notification',
+                'params': {
+                    'message': _('You already have defined a Stripe Webhook.'),
+                    'sticky': False,
+                    'type': 'danger',
+                }
+            }
+        if 'localhost' in self.company_id.get_base_url():
```

Comment:
I wouldn't bother handling this case. Only developers will get that message.

Reply:
Isn't it better to be explicit ?

Reply:
Thinking about it twice, I'll just leave a comment there to explain that it only works with public URL.

Reply:
Works for me :+1: 

Many integrations only work with HTTPS. When a developer receives a suspicious error message, their next move if usually to set up an HTTP tunnel.